src/mol-plugin-ui/sequence.tsx

```diff
@@ -389,13 +401,14 @@ export class SequenceView extends PluginUIComponent<{ defaultMode?: SequenceView
                 {params.operator.options.length > 1 && <>
                     <PureSelectControl title={`[Instance] ${PD.optionLabel(params.operator, values.operator)}`} param={params.operator} name='operator' value={values.operator} onChange={this.setParamProps} />
                 </>}
+                {<BoolControl name={`[Secondary Structure]`} param={SecondaryStructureViewModeParam} value={values.secondaryStructure} onChange={({ value }) => { this.state.secondaryStructure = value; }}/>}
```

Comment:
Need to call `setState` instead of just re-assigning it. The original assigment is just to provide defaults.

```suggestion
                {<BoolControl name={`[Secondary Structure]`} param={SecondaryStructureViewModeParam} value={values.secondaryStructure} onChange={({ value }) => this.setState({ secondaryStructure: value }) }/>}
```